### PR TITLE
Adds bone "tongue" to skeletons

### PR DESCRIPTION
**Pull request by coiax**
image

> Apparently skeletons alter the sounds they produce through oscillation of their teeth, hence their characteristic rattling.
- [x] Work out how to change a users spans based on their tongue

:cl: coiax
tweak: Skeletons now have bone "tongues"
/:cl:

### CODE REVIEW

**Comment by Mercenaryblue** [comment]
I'm split. Realistically, skeletons don't have tongue, the operation isn't the same (tongue vs teeth implant), players already have teeth anyway.

But... rattle 'tongue' object off skeletons. It sounds too funny not to be.

**Comment by coiax** [comment]
I was originally just going to put in a bone coloured tongue until someone just suggested they have comedy chattering teeth, and that was too good an idea to pass up.

**Comment by Mercenaryblue** [comment]
Idea: If skeletons gets decapitated, the skull should be able to talk and crawl

**Comment by coiax** [comment]
@phil235 please implement skeleton skulls being independent k thnx

File: code/modules/surgery/organs/organ_internal.dm [hunk 1]
```diff
@@ -358,6 +361,43 @@
...
+	desc = "Apparently skeletons alter the sounds they produce \
+		through oscillation of their teeth, hence their characteristic \
+		rattling."
+	icon_state = "tonguebone"
+	say_mod = "rattles"
+	attack_verb = list("bitten", "chattered", "chomped", "enamelled", "boned")
+
+	var/chatter = FALSE
+	var/phomeme_type = "sans"
+	var/list/phomeme_types = list("sans", "papyrus")
```
Comment (Core0verload) [inline code/modules/surgery/organs/organ_internal.dm]: another undermeme?

Comment (coiax) [inline code/modules/surgery/organs/organ_internal.dm]: Look at the diffs, this is a refactor of existing features, nothing more.

File: code/modules/surgery/organs/organ_internal.dm [hunk 2]
```diff
@@ -358,6 +361,43 @@
...
+/obj/item/organ/tongue/bone/New()
+	. = ..()
+	phomeme_type = pick(phomeme_types)
+
+/obj/item/organ/tongue/bone/TongueSpeech(var/message)
+	. = message
+
+	if(chatter)
+		//Annoy everyone nearby with your chattering.
+		return
```
Comment (phil235) [inline code/modules/surgery/organs/organ_internal.dm]: this does nothing.

Comment (coiax) [inline code/modules/surgery/organs/organ_internal.dm]: hooks for future implementation.

File: code/modules/surgery/organs/organ_internal.dm [hunk 3]
```diff
@@ -358,6 +361,43 @@
...
+
+	var/chatter = FALSE
+	var/phomeme_type = "sans"
+	var/list/phomeme_types = list("sans", "papyrus")
+
+/obj/item/organ/tongue/bone/New()
+	. = ..()
+	phomeme_type = pick(phomeme_types)
+
+/obj/item/organ/tongue/bone/TongueSpeech(var/message)
```
Comment (phil235) [inline code/modules/surgery/organs/organ_internal.dm]: the var/ isn't needed

**Pull request merged**